app/assets/javascripts/components/students/students_filters.es6.jsx

```diff
@@ -1,6 +1,14 @@
 class StudentsFilters extends Component {
 
   // --------------------------------------------------
+  // Setup
+  // --------------------------------------------------
+  constructor(props) {
+    super(props);
+    this._listener = (state) => this.setState(state);
+  }
```

Comment:
Instead of setting this component to listen to the store, notice that it's parent - the `StudentsPage` - already does! That means the parent can just pass this component down props instead of having another component listen to the store.